File: PostTrainingActivities/RestaurantAssignment/Models/Restaurant.cs
using System;
using System.Collections.Generic;

namespace RestaurantAssignment.Models;

public partial class Restaurant
{
    public int RestaurantId { get; set; }

    public string? RestaurantName { get; set; }

    public string? Grade { get; set; }

    public string? Street { get; set; }

    public string? City { get; set; }

    public string? State { get; set; }

    public virtual ICollection<RestaurantCuisine> RestaurantCuisines { get; } = new List<RestaurantCuisine>();

    public virtual ICollection<RestaurantMenu> RestaurantMenus { get; } = new List<RestaurantMenu>();

    public virtual ICollection<Score> Scores { get; } = new List<Score>();
}

File: PostTrainingActivities/RestaurantApp/src/app/classes/restaurant.ts
export class Restaurant {
    public RestaurantId: number;
    public RestaurantName: string;
    public Grade: string;
    public Street: string;
    public City: string;
    public State: string;

    constructor(restaurantId: number, restaurantName: string, grade: string, street: string, city: string, state: string) {
        this.RestaurantId = restaurantId;
        this.RestaurantName = restaurantName;
        this.Grade = grade;
        this.Street = street;
        this.City = city;
        this.State = state;
    }
}
